Doc/library/configparser.rst

```diff
@@ -1088,10 +1091,10 @@ ConfigParser Objects
       given *section*.  Optional arguments have the same meaning as for the
       :meth:`get` method.
 
-   .. versionchanged:: 3.8
-      Items present in *vars* no longer appear in the result.  The previous
-      behaviour mixed actual parser options with variables provided for
-      interpolation.
+      .. versionchanged:: 3.8
```

Comment:
It looks like you are addressing another problem here, one that is only in master.  With this change, it won't be possible to auto backport this fix to 3.7.  Suggest you include this in another PR if necessary.

Reply:
Yes, exactly. I'll update the PR.